app/src/main/java/com/example/labparticipation/Star.java

```diff
@@ -0,0 +1,2 @@
+package com.example.labparticipation;public class Star {
```

Comment:
Add newline before `public class`

Reply:
oh weird, it looks fine in Android Studio 
<img width="349" alt="Screen Shot 2023-01-31 at 5 54 34 PM" src="https://user-images.githubusercontent.com/70451233/215918453-1baf2479-c5eb-4af0-9fe8-3e0b07d12143.png">

Reply:
nvm should be good for a re-review now lol